src/pages/citations.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react"
+import { graphql } from "gatsby"
+import Layout from "../components/layout"
+import SEO from "../components/seo"
+import QuotationList from "../components/quotationListItem"
+import LayoutSidebar from "../components/layoutSidebar"
+
+interface QuotationPageProps {
+  data: {
+    allPrismicQuotation: {
+      nodes: [] // it should be something like PrismicBlogPost[] insteead of a simple []
+    }
+  }
+}
+
+const QuotationPage = ({ data }: QuotationPageProps) => {
+  const quotation = data.allPrismicQuotation.nodes
+
+  return (
+    <Layout withInstagram={false}>
+      <SEO title="Un podcast sur l’histoire des femmes dans le monde artistique présenté par Aldjia" />
```

Comment:
you should find a better title 

Example
![image](https://github.com/flexbox/artaufeminin/assets/360936/6005bc7f-02c3-4f99-85c6-846bf5c7a986)